components/maltego-data-capture-form/index.js

```
@@ -0,0 +1,202 @@
+  }
+
+  const renderMessage = () => {
+    if (formState === "success") {
+      return (
+        <div
+          className="dcf-message dcf-message-success"
+          style={{
+            color: successFontColor || undefined,
+            backgroundColor: successBgColor || undefined,
+          }}
+          dangerouslySetInnerHTML={{ __html: successFormMessage }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)